storage/engine/rocksdb_test.go

```diff
@@ -484,6 +494,40 @@ func BenchmarkMVCCMergeTimeSeries(b *testing.B) {
 	runMVCCMerge(&value, 1024, b)
 }
 
+func runMVCCDeleteRange(valueBytes int, b *testing.B) {
+	// 512 KB ranges so the benchmark doesn't take forever
+	const rangeBytes = 512 * 1024
+	const overhead = 48 // Per key/value overhead (empirically determined)
+	numKeys := rangeBytes / (overhead + valueBytes)
+
+	b.SetBytes(rangeBytes)
+	b.ResetTimer()
+	b.StopTimer()
+
+	for i := 0; i < b.N; i++ {
+		rocksdb, stopper := setupMVCCData(1, numKeys, valueBytes, false, b)
```

Comment:
Continually recreating the data is likely to be slow. I think you get rid of the changes to `setupMVCCData` and instead copy the directory and its contents on each loop iteration.